consensus/progpow/consensus.go

```diff
@@ -652,6 +652,8 @@ func (progpow *Progpow) Prepare(chain consensus.ChainHeaderReader, header *types
 func (progpow *Progpow) Finalize(chain consensus.ChainHeaderReader, header *types.WorkObject, state *state.StateDB) {
 	nodeLocation := progpow.NodeLocation()
 	nodeCtx := progpow.NodeLocation().Context()
+	// Accumulate any block and uncle rewards and commit the final state root
+	progpow.accumulateRewards(chain.Config(), state, header)
```

Comment:
where was the previous accumulateRewards function? Looks like this is an addition